src/nsidc/metgen/metgen.py

```diff
@@ -147,6 +72,13 @@ def init_config(configuration_file):
     cfg_parser.set(DESTINATION_SECTION_NAME, "local_output_dir", Prompt.ask("Local output directory", default="output"))
     cfg_parser.set(DESTINATION_SECTION_NAME, "ummg_dir", Prompt.ask("Local UMM-G output directory (relative to local output directory)", default="ummg"))
     cfg_parser.set(DESTINATION_SECTION_NAME, "kinesis_arn", Prompt.ask("Kinesis Stream ARN"))
+    cfg_parser.set("Destination", "write_cnm_file", Prompt.ask("Write CNM messages to files (True/False)"))
```

Comment:
Hmmm, just realized that I put my thoughts about the boolean "write to files?" value in my comments and tests in #36 rather than here. All that rebasing excitement led to me forgetting which issue addressed what problem. I think the correct answer is to treat some of my feedback for #36 as part of this code review -- do you want me to move comments/code/tests from `issue-36` to the `issue-38` branch?

Reply:
I'll do them as part of 36